styles.css

```diff
@@ -264,3 +264,85 @@ button:disabled {
   color: #396df2;
   width: 50%;
 }
+
+/* contact form styling */
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  border-top-left-radius: 80px;
+  background: #6070ff;
+  align-items: center;
+  width: 22vh 8vw;
+}
+
```

Comment:
- [ ] Please consider adding a `background-image` for the contact section to get a design close to the Figma template as much as possible 💪 This is in accordance with the project requirements ⛏🤓

  > _You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)._


  Your design | Given design
  ---| ---|
  ![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/194a74b4-3284-46c9-8a18-c210bdebc525) | ![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/9fa3ccbf-2e8c-4302-a875-bc6657b18c85)